scripts/extract_variables_from_integration_tests.py

```diff
@@ -0,0 +1,92 @@
+"""
+extracts manageiq integration test provider data/credentials
+into variables files for the wrapanapi testsuite
+
+this script uses manageiq integration test domain knowledge
+and should be executed using its virtualenv
+"""
+
+import click
+import six
+import pathlib2
+import json
+
+
+WANTED_TAGS = frozenset(['rhevm', 'ms_scvmm'])
+
+
+def extract_rhevm_system(mgmt, credentials):
+    system_credentials = credentials[mgmt.credentials]
+    system = dict(
+        hostname=mgmt.hostname,
+        version=mgmt.version,
+        username=system_credentials.username,
+        password=system_credentials.password,
+    )
+    return dict(
+        system=system,
+        templates=mgmt.templates
+    )
+
+
+def extract_scvmm_system(mgmt, credentials):
+    system_credentials = credentials[mgmt.credentials]
+    system = dict(
+        hostname=mgmt.hostname,
+        username=system_credentials.username,
+        password=system_credentials.password,
+        domain=system_credentials.domain,
+        provisioning=mgmt.provisioning
+
+    )
+    return dict(
+        system=system,
+        templates=mgmt.templates,
+    )
+
+
+system_type_map = {
+    'scvmm': extract_scvmm_system,
+    'rhevm': extract_rhevm_system,
+}
+
+
+def extract_system(mgt, credentials):
```

Comment:
**OPTIONAL:** Add a doc string.  I would like one but mainly for you to describe what a system is that is being extracted.